### DISCUSSION TITLE
Who has awesome holiday lights powered by NightDriverStrip?

### DISCUSSION THREAD

**Discussion by robertlipe** [PlummersSoftwareLLC/NightDriverStrip#502] [General]
This could be a request in show and tell. It could also be "Project X has a really awesome (license-compatible) holiday display and I wish NDS had it" But we're into the[ time of year](https://www.youtube.com/watch?v=e2I-_tIDV-4&ab_channel=RyanHill) where massive lighting is a public thing and our community should be the heroes of that world.

- Who has a really awesome NightDriverLED display?
- Who has a really awesome display on something with source available that's license-compatible that "just" needs porting to NightDriverLED?
- Who has a raw idea that they can pencil out enough in enough detail for a programmer to make it happen for the holidays?

To prime the pump, I may try to port Kriegsman's Holiday [Twinklefox](https://gist.github.com/kriegsman/756ea6dcae8e30845b5a) and/or DemoReel100_ChristmasSparkles' christmasSparkles()/christmasSparklesRG() to get the ball rolling. They're both for FastLED, the library we use for strip, so they're probably not too far off once the scheduling differences are fixed.

But I'm interested in hearing what's NEW or what you all have planned for YOUR displays in this season when your spouse asks for a demonstration of where the household LED budget is going. :-)

**Comment by mikejohnau** [comment]
Ok, as nobody else is willing... Nothing special. But I live on a battle axe block ie, way back from the street behind other houses so no need for curb appeal at all, whatever the time of year. Plus my wife and I will be overseas for christmas visiting her family in another country. Absolutely no need for dasblinkenxmaslights..... But I couldn't resist...

Uploading xmas_blinken2.mov…

**Comment by KeiranHines** [comment]
I really like the system Matt Parker made back in 2021 (https://youtu.be/WuMRJf6B5Q4). It even made it to be part of a Harvard course. https://github.com/GSD6338/XmasTree. Not saying NDS should support the idea, just that I really like the idea.

**Reply by robertlipe** [reply]
Cool idea. Given that my other [project](www.gpsbabel.org) involves GPS tracks that often involves removing 'zingers' where you'll seem to fly to cuba and back in a second before resuming your bike ride, [reducing the number of points in a track](https://www.gpsbabel.org/htmldoc-development/filter_simplify.html) while retaining overall fidelity, and kalman filters, I saw where that was heading before he said the words aloud.
